src/clusterfuzz/_internal/swarming/__init__.py

```diff
@@ -93,13 +95,14 @@ def _get_new_task_spec(command: str, job_name: str, download_url: str):
                       cas_instance=cas_instance,
                       digest=swarming_pb2.Digest(
                           hash=digest_hash, size_bytes=digest_size_bytes)),
-                  execution_timeout_secs=86400,
+                  execution_timeout_secs=execution_timeout_secs,
                   env=[
+                      swarming_pb2.StringPair(key='UWORKER', value='True'),
+                      swarming_pb2.StringPair(key='SWARMING_BOT', value='True'),
                       swarming_pb2.StringPair(
-                          key='UWORKER_INPUT_DOWNLOAD_URL', value=download_url),
-                      swarming_pb2.StringPair(key='UWORKER', value=True),
-                      swarming_pb2.StringPair(key='SWARMING_BOT', value=True)
-                  ]))
+                          key='DOCKER_IMAGE', value=docker_image)
```

Comment:
Is this environment variable required? If not, maybe we should skip specifying it when `instance_spec['docker_image']` is None.